dask/dataframe/tests/test_dataframe.py

```diff
@@ -1549,6 +1565,18 @@ def test_corr():
     pytest.raises(TypeError, lambda: da.corr(ddf))
 
 
+def test_cov_corr_meta():
+    df = pd.DataFrame({'a': np.array([1, 2, 3]),
+                       'b': np.array([1.0, 2.0, 3.0], dtype='f4'),
+                       'c': np.array([1.0, 2.0, 3.0])},
+                       index=pd.Index([1, 2, 3], name='myindex'))
+    ddf = dd.from_pandas(df, npartitions=2)
+    eq(ddf.corr(), df.corr())
+    eq(ddf.cov(), df.cov())
+    assert ddf.a.cov(ddf.b)._meta.dtype == 'f8'
+    assert ddf.a.corr(ddf.b)._meta.dtype == 'f8'
```

Comment:
I believe only the default integer width changes on each platform, the default float should be `float64` in all cases:

> int_ 	Default integer type (same as C long; normally either int64 or int32)
...
float_ 	Shorthand for float64.

http://docs.scipy.org/doc/numpy-1.10.1/user/basics.types.html#array-types-and-conversions-between-types